core/src/main/res/values/strings.xml

```diff
@@ -449,6 +449,12 @@
     <string name="pref_playback_speed_title">Playback Speeds</string>
     <string name="pref_playback_speed_sum">Customize the speeds available for variable speed audio playback</string>
     <string name="pref_feed_playback_speed_sum">The speed to use when starting audio playback for episodes in this podcast</string>
+    <string name="pref_feed_skip">Auto Skip</string>
+    <string name="pref_feed_skip_sum">Skip introductions and ending credits.</string>
+    <string name="pref_feed_skip_ending">Skip last</string>
+    <string name="pref_feed_skip_intro">Skip first</string>
+    <string name="pref_feed_skip_ending_toast">Skip last %d %s</string>
```

Comment:
I would use `Skipped` instead of `Skip` in the Toasts because they show something that happened, not an action that can be done.